pkg/gcv/configs/config.go

```diff
@@ -109,7 +109,7 @@ func asConstraintTemplate(data *UnclassifiedConfig) (*ConstraintTemplate, bool)
 	if err != nil {
 		return nil, false // field expected to exist
 	}
-	rego, err := data.Yaml.GetPath("spec", "targets", expectedTarget, "rego").String()
```

Comment:
Not directly related to this CL, but adding debug info lines in these methods (get as template/constraint) and log the error reason that the conversation failed. 

I believe there is a variable that defines a verbosity level for these style debug logs already.

Reply:
SG. I'll do that in a follow-up change.